pyproject.toml

```diff
@@ -46,10 +46,10 @@ dependencies    = [
     "cryptography==41.0.7",
     # pyOpenSSL 23.2.0 is required to work with cryptography 41+
     "pyOpenSSL==23.2.0",
-    "orjson==3.9.12",
+    "orjson==3.9.10",
     "packaging>=23.1",
     "pip>=21.3.1",
-    "python-slugify==8.0.1",
+    "python-slugify==4.0.1",
```

Comment:
Please don't modify these

Reply:
I modified the recipes and made a new commit